### DISCUSSION TITLE
Typing a book

### DISCUSSION THREAD

**Discussion by Aadam-Marshall** [monkeytypegame/monkeytype#5570] [Ideas] [answered]
It would be very cool to be able to type books on Monkeytype. I'm tired of typing various quotes (sometimes I even stumble upon even quotes that are abhorrent and bring me negative feelings) and also wanted to read a book, so why not read and type what you like at the same time? And add a bookmark so you can start typing from where you left off, maybe some other essential features, etc. No account is needed for that, of course, just save the info as cookies.

www.typelit.io does it, but is pay-walled.

**Comment by fehmer** [comment, marked as answer]
- choose "custom" in test settings
- click "change"
- click "open file" and load your book
- click "save" and check "Long text  ibook mode)

**Reply by Aadam-Marshall** [reply]
Yeah sorry, I've just now noticed it.
